package.json

```diff
@@ -23,8 +23,8 @@
     "verdaccio-publish": "yarn verdaccio-clean && yarn publish-to-verdaccio",
     "verdaccio-disconnect": "source .circleci/local_publish_helpers.sh && unsetNpmRegistryUrl",
     "verdaccio-stop": "kill -9 $(lsof -n -t -iTCP:4873 -sTCP:LISTEN)",
-    "setup-dev": "rm -rf yarn.lock && (yarn && lerna run build) && yarn add-cli-no-save && (yarn hoist-cli && yarn rm-dev-link && yarn link-dev)",
-    "add-cli-no-save": "yarn add @aws-amplify/amplify-category-api @aws-amplify/cli-internal -W && git restore package.json yarn.lock",
```

Comment:
Did you verify that w/ these changes `setup-dev` still behaves as expected?

Reply:
Yes I just verified locally. The new script works well.